sampleModels/README.md

```diff
@@ -1,4 +1,4 @@
-Sample models are provided in as many of the following formats as possible:
+bSample models are provided in as many of the following formats as possible:
```

Comment:
Good catch. And I think that's a good idea for the READMEs, I'll add that when I get a chance.